yarn-project/circuits.js/src/abis/abis.test.ts

```diff
@@ -190,13 +188,6 @@ describe('abis', () => {
     expect(emptyHash).toMatchSnapshot();
   });
 
-  it('Computes an empty public inputs hash ', () => {
```

Comment:
How come we are not doing this? 👀  
The current implementation is not computing the single public inputs hash, so I know it is kinda unused currently, but it should be passed in when we are actually verifying the proofs in contract. 

Edit: Oh, just moved inwards.